src/mudsys/main.351

```diff
@@ -2023,7 +2023,7 @@ GCPDL:	-GCPLNT,,GCPDL
 
 PURE
 
-MUDSTR:	ASCII /MUDDLE /
+MUDSTR:	ASCII /MUDDLE ��
```

Comment:
What is this character?

Reply:
Oh I understand now, actually it's `/MUDDLE ^?^?^?/`.

Reply:
In the GITHUB repository we will find in https://github.com/PDP-10/its/blob/master/src/mudsys/main.353 in line 2030:
MUDSTR:	ASCII /MUDDLE ¯
STRNG:	-1
	-1
	-1
	ASCIZ / IN OPERATION./
But ITS-TAR makes this in ITS:
MUDSTR:	ASCII /MUDDLE ^?^?^?/ 
(where ^? = 177/DEL)
That was something I stumbled over while analysing the muddle build's patching of the muddle version number.

Reply:
```
mkdir "mudsav"
respond "*" ":stinkm\r"
respond "STINK." "MMUD56 STINK\033@\033\033"
expect "SETPUR"
respond "\n" "D\033\033"
respond "\n" "strng/\0331\"56^?^?^?\033\r"
respond "\n" ":pdump mudsav; ts mud56\r"
respond "*" ":start\r"
respond "..PERM/ -1" ":pdump mudsav; ts mdl56\r"
respond "*" ":kill\r"
```